addons-l10n/en/scratch-messaging.json

```diff
@@ -15,7 +15,7 @@
   "scratch-messaging/chars-left": "{num, plural, one {1 character left} other {# characters left}}",
   "scratch-messaging/follows": "Follows",
   "scratch-messaging/stMessages": "Messages from the Scratch Team",
-  "scratch-messaging/stMessagesConfirm": "Are you sure you want to dismiss this message from the Scratch Team? It cannot be recovered.",
+  "scratch-messaging/stMessagesConfirm": "Dismiss this message?",
```

Comment:
I like the shorter question, but I think the second sentence should stay - it makes it easier to understand why there's a confirmation.

Reply:
What if it said "delete this message"?

Reply:
Maybe "Delete this message permanently?"